src/shainet/basic/exprimental.cr

```diff
@@ -39,7 +39,7 @@ module SHAInet
       end
 
       unless stealth # Hide report during training
-        @input_layers.last.activations.show(@logger, "input layer activations:")
+        @input_layers.last.activations.show(LOG, "input layer activations:")
```

Comment:
I don't think there's a need to pass a `Log` instance here anymore.
ditto elsewhere

Reply:
I have no idea what the `#show` method does. 

Should I just remove the first argument or the method call altogether ?

Reply:
I'd remove the argument.